### ISSUE TITLE
"Sorry. Unable to access that page" after using Crudss generator to create jobs collection

### ISSUE THREAD

**Issue by pdebaets** [radiegtya/meteoris#64]
Hi,

I'm following the tutorial video here:

3 Meteoris Crudss Generator
https://www.youtube.com/watch?v=mqmoJCMJzEg&index=3&list=PLkq0XDlLyPbA6y51gLeJSnK419PFFgS5M

After I generate the "jobs" collection with the one field named "title", I receive the "Success generating code!" message.  I then wait, and enter "http://localhost:3000/jobs" in the address bar of Chrome. I get the message "Sorry. Unable to access that page". I'm using chrome on Windows 8.1.

when I search through my project (I'm using webstorm), the only place where the string "jobs" is found is in a readme.md file and in folder tests\cucumber\features. It's as if  the code was never generated...?

Can you help?

Thanks,

Peter De Baets

**Comment by pdebaets** [comment]
When building the code, the statement

this.write(path, content);

... seems to be picking up the wrong default path. Here's the error from the server terminal:

I20150507-13:19:24.871(-7)? { [Error: ENOENT, open 'F:\Users\Peter\Documents\Apps Meteor\ledger\.meteor\local\build\programs\server\undefined\client\routers\jobs.js']
I20150507-13:19:24.871(-7)?   errno: 34,

Can you help?

**Comment by pdebaets** [comment]
Looks like it is some kind of a windows issue. I think the

process.env.PWD

... statement in MugenServer.js does not return the correct path in Windows environments. When I replace the statement above with a hard-coded path, the code generation works.

![image](https://cloud.githubusercontent.com/assets/8390093/7525697/95c99970-f4c0-11e4-80f2-9236b6111390.png)

**Comment by radiegtya** [comment]
yeah, we currently not supporting windows this time, and ofc You will find another problem if You are developing advance meteor using windows. We'll fixing it when developing in windows is stable enough. 

Anyway, thanks for the report

**Comment by pdebaets** [comment]
What's the other problem that I will find if I am developing advanced meteor using windows?

**Comment by radiegtya** [comment]
the other problem is you can't developing mobile apps.

**Comment by pdebaets** [comment]
I see that you've closed this issue, but isn't it still a problem? Other windows users might come along and get stuck like I did. 

Maybe I'm not used to the way that github works, but shouldn't you either save this issue for a future fix, or take some action like including the text "Windows not supported" in the readme file, or include a link to the hardcoded workaround above?

**Comment by radiegtya** [comment]
Good idea @pdebaets. I'll add "Windows not supported" in documentation and in command line, then close the issue.

**Comment by xyshift** [comment]
I didn't see a "windows not supported" in the docs, and I'm having the same trouble (also added a comment with the errors I'm receiving on someone else's issue - but they are seeing it on Ubuntu, so it's not just Windows having the problem). 

Would this be do-able if it was run in a bash shell?
